src/components/cards/SocialCard.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { View, Text, Pressable, Linking } from 'react-native';
+
+export type Social = {
+  type: string;
```

Comment:
You can declare an `enum` representing the possible values for the `type` field:

```ts
enum SocialTypes {
  Discord,
  Website,
  Facebook,
  Instagram,
  Twitter,
}
```

Then set the `type` to be any of the keys from that enum:
```suggestion
  type: keyof SocialTypes;
```